llvm/include/llvm/ProfileData/MemProf.h

```diff
@@ -44,6 +44,13 @@ enum class Meta : uint64_t {
 
 using MemProfSchema = llvm::SmallVector<Meta, static_cast<int>(Meta::Size)>;
 
+// Returns the full schema currently in use.
+MemProfSchema getFullSchema();
+
+// Returns the schema consisting of the fields currently consumed by the
```

Comment:
Lets make this comment a little more future proof, how about "Returns the schema consisting of the fields used for hot cold memory hinting."